files/libs/xroad_libs.sh

```diff
@@ -5,6 +5,8 @@
 # e.g.
 #   . ./xroad_libs.sh
 
+Q_CERTS_FOLDER=/etc/xroad/signer/
```

Comment:
What's the Q_ prefix?

Reply:
aa :D, I wanted to discuss this - at the moment there is PX_ prefix that stands for PlanetCross, yes? So Q_ would be for ConneQt, or?

Reply:
I see. Then let's keep the Q_. When we have a major upgrade let's replace all the PX_ to Q_.